src/server/v2.0/route/legacy.go

```diff
@@ -62,8 +62,6 @@ func registerLegacyRoutes() {
 	beego.Router("/api/"+version+"/system/CVEWhitelist", &api.SysCVEWhitelistAPI{}, "get:Get;put:Put")
 	beego.Router("/api/"+version+"/system/oidc/ping", &api.OIDCAPI{}, "post:Ping")
 
-	beego.Router("/api/"+version+"/logs", &api.LogAPI{})
```

Comment:
Please remove the router of getting project log as well

Reply:
done